user_guide_src/source/incoming/incomingrequest.rst

```diff
@@ -253,10 +253,14 @@ and uses best practices to minimize any security risks.
 		echo $file->getType();          // image/jpg
 	}
 
-You can also retrieve a single file based on the filename given in the HTML file input::
+You can retrieve a single file uploaded on its own, based on the filename given in the HTML file input::
 
 	$file = $request->getFile('uploadedfile');
 
+You can retrieve a single file uploaded as part of multiple ones, based on the filename given in the HTML file input::
+
```

Comment:
As `getFileMultiple()` return an array with the files, the text should be something like this:
You can retrieve the list of the files uploaded as multiple ones, based on the filename given in the HTML file input::
(please correct my english :)  if its wrong)

Reply:
I've adjusted the user guide text for both cases you point out, and that will be in the next push.